taxi/models.py

```diff
@@ -19,6 +19,7 @@ class Driver(AbstractUser):
     class Meta:
```

Comment:
DB is still here. remove from the cache and then push on GitHub
![image](https://user-images.githubusercontent.com/95351527/202919568-7f432f09-35fd-441d-a66f-7338ff733268.png)

Reply:
sorry, i remove the db.